app/hackathons/[hackathonSlug]/features/info/info.tsx

```diff
@@ -47,7 +47,7 @@ function Sponsors({ sponsors }: { sponsors: TInfo.Sponsor[] }) {
     return (
       <InfoDropdown
         targetLabel={
-          <Translate token={"v2.pages.hackathons.details.info.countSponsors"} params={{ count: sponsors.length }} />
+          <Translate token={"v2.pages.hackathons.details.info.countPrograms"} params={{ count: sponsors.length }} />
```

Comment:
```suggestion
          <Translate token={"v2.pages.hackathons.details.info.countPrograms"} params={{ count: programs.length }} />
```

don't know if we need this change

Reply:
we save this in hackathons for now